main.tf

```diff
@@ -44,92 +52,104 @@ resource "aws_db_subnet_group" "this" {
   tags = var.tags
 }
 
+################################################################################
+# Cluster
+################################################################################
+
 resource "aws_rds_cluster" "this" {
   count = local.create_cluster ? 1 : 0
 
-  # Notes:
```

Comment:
lots of noise due to the re-ordering to match the AWS provider (makes it easier to compare what we support vs what we need to add), apologies for that. However, nothing was deleted - only re-order or the addition of new arguments that we did not previously support (mostly for RDS multi-AZ plus a few others)